.env

```diff
@@ -4,4 +4,4 @@ AWS_SECRET_ACCESS_KEY=verySecretKey1
 AWS_REGION='us-east-1'
 AWS_S3_FORCE_PATH_STYLE='true'
 DRIVER=chromedriver
-DISPLAY=:15
```

Comment:
The .env file was committed by accident - the idea of the .env file is for people to customise to their environment - hence why you wanted 2 and I wanted 15 lol